components/HomeProducts.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect} from 'react'
+import { Flex, Grid, Box, Text} from '@chakra-ui/react'
+import Product from './Product'
+
+function HomeProducts({coffees, itemsLeft}) {
```

Comment:
this could be just products and setProducts

Reply:
Just to be sure. Are you referring to properties here?